eclipse-collections-code-generator/src/main/resources/api/map/mutableObjectPrimitiveMap.stg

```diff
@@ -111,6 +111,8 @@ public interface MutableObject<name>Map\<K> extends Object<name>Map\<K>
      */
     <type> getIfAbsentPut(K key, <type> value);
 
+    <type> getAndPut(K key, <type> defaultValue, <type> putValue);
```

Comment:
Order is important here because we have two parameters with the same primitive type.

If I were going to guess at the order without looking, I would have expected 2 and 3 to be reversed.

1. The key
2. The new value
3. The default value to use if necessary

Reply:
Sounds good. Done.